uniffi_macros/src/enum_.rs

```diff
@@ -21,7 +21,7 @@ pub fn expand_enum(input: DeriveInput, module_path: Vec<String>) -> TokenStream
             let write_match_arms = variants.iter().enumerate().map(|(i, v)| {
                 let v_ident = &v.ident;
                 let fields = v.fields.iter().map(|f| &f.ident);
-                let idx = Index::from(i);
+                let idx = Index::from(i + 1);
```

Comment:
"There are 2 hard problems in computer science: cache invalidation, naming things, and off-by-1 errors."